image-sync-formula/image-synchronize/sync.sls

```diff
@@ -51,6 +51,7 @@ images:{{ image_id }}:{{ image_version }}:
 {%-     set local_path_relative = image_data['sync'].get('local_path') %}
 {%-     set local_path = rootdir + '/' + local_path_relative %}
 {%-     set local_file = local_path + '/' + name %}
+{%-     set local_bundle_cache = salt["cp.is_cached"](image_data['sync'].get('bundle_url', "missing")) %}
```

Comment:
When `bundle_url` is not present then `local_bundle_cache` is set to "missing".

But then later it just checks `if local_bundle_cache` but when it is missing, this condition will evaluate as `true`. I don't think this is intended.